qwik-graphql-tailwind/src/components/user-repos/user-repos.tsx

```diff
@@ -24,7 +27,7 @@ export const UserRepos = component$(({ repos, owner }: UserReposProps) => {
   return (
     <>
       <RepoFilters languages={languages} resultCount={repos.nodes.length} />
-      {filteredAndSortedRepos.map(
+      {languageFilter.map(
```

Comment:
why we are iterating over the languageFilter array, we should use `filteredAndSortedRepos`, please make that change